Servers/UI/OJS.Servers.Ui/ClientApp/src/common/filter-utils.ts

```diff
@@ -12,9 +12,10 @@ const getNextIdGen = function* () {
 
 const getId = getNextIdGen();
 
-const generateFilterItems = (type: FilterType, ...names: string[]) => names.map((name) => ({
-    name,
+const generateFilterItems = (type: FilterType, ...filters: FilterInfo[]) => filters.map(({ name, value }) => ({
+    name: name,
     type,
+    value: value,
```

Comment:
These should be `eslint` errors.
No need for `value: value`, just `value` is good enough